### PR TITLE
add favicons

### PR DESCRIPTION
**Pull request by macv93**

### CODE REVIEW

File: public/site.webmanifest [hunk 1]
```diff
@@ -0,0 +1 @@
+{"name":"","short_name":"","icons":[{"src":"/android-chrome-192x192.png","sizes":"192x192","type":"image/png"},{"src":"/android-chrome-512x512.png","sizes":"512x512","type":"image/png"}],"theme_color":"#ffffff","background_color":"#ffffff","display":"standalone"}
```
Comment (mrozbarry) [inline public/site.webmanifest:1]: I don't need this file, let's remove it.

**Review by mrozbarry — CHANGES_REQUESTED** [review]
The favicon should be added to public/index.html and public/timer.html .

Also, since I primarily use the stop watch font awesome icon, what do you think about:
https://favicon.io/emoji-favicons/stopwatch

That includes instructions for the link tags that will be necessary to add to the two html files.

File: public/site.webmanifest (hunk 1 above)
Comment (mrozbarry) [reply public/site.webmanifest:1]: Hmm, actually, keep it, this would be handy for phone shortcuts.

Comment (macv93) [reply public/site.webmanifest:1]: Hello @mrozbarry, the favicons have been updated and the missing headers were added to public/timer.html

Comment (mrozbarry) [reply public/site.webmanifest:1]: Can you set name and short_name to `"mobtime"` please and thanks

Comment (macv93) [reply public/site.webmanifest]: Yes sir, I can definitely do that

**Pull request merged**